src/style.css

```diff
@@ -1,3 +1,60 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.img_logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav_list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_link_item {
+  display: inline-block;
+  margin: 0 10px;
```

Comment:
better to give `margin-right` or `margin-left`, and remove margin from last or first child instead of giving `first-item` as you did [here](https://github.com/mate-academy/layout_moyo-header/pull/3620/files#diff-1104bdfdc4d3b01a85d7af489598121125db6bc016db2a48a083ef654c97b0f5R54)